.github/dependabot.yml

```diff
@@ -15,3 +15,8 @@ updates:
   schedule:
     interval: weekly
   target-branch: dependencies
+- package-ecosystem: pip
+  directory: "/app/tests"
+  schedule:
+    interval: weekly
+  target-branch: dependencies
```

Comment:
Nice catch. I think the only one that might not have be included is the root app/package.json (which currently doesn't have any dependencies or devdependencies, so hard meh):

```sh
git ls-files | grep -E "requirements|package"

app/experimenter/static/core/package.json
app/experimenter/static/rapid/package.json
app/package.json
app/requirements/constraints.txt
app/requirements/default.txt
app/tests/requirements.txt
```